website/docs/cloud-docs/registry/publish-modules.mdx

```diff
@@ -151,3 +151,7 @@ Terraform Cloud does not typically allow one organization's workspaces to use pr
 Instead, you can share modules across organizations by sharing the underlying VCS repository. Grant each organization access to the module's repository, and then add the module to each organization's registry. When you push tags to publish new module versions, both organizations update accordingly.
 
 Terraform Enterprise administrators can configure [module sharing](/terraform/enterprise/admin/application/registry-sharing) to allow organizations to use private modules from other organizations.
+
+## Generating Module Tests (Beta)
+
+You will be able to generate tests for your module once a version has been published. Tests can only be generated once per module and are intended to be reviewed by the module's authors before being checked into version control. If the module's configuration files exceed 128KB in total size, Terraform Cloud will not be able to generate tests for that module. You must have [permission to manage registry modules](/terraform/cloud-docs/users-teams-organizations/permissions#manage-private-registry) and the [module test generation setting enabled](/terraform/cloud-docs/users-teams-organizations/permissions#manage-private-registry) in order to generate tests.
```

Comment:
- I suggest putting a bit of lead-in context here — one of the things that rarely gets stated with this project is that these tests are meant to be run with `terraform test`.
- Wrong anchor link on the second one there.
- This is a permission, not a setting, right? Or is there an additional org-wide setting I'm forgetting about? 

```suggestion
You can generate tests for your module, and can run the resulting tests with [the `terraform test` command](/terraform/cli/commands/test).

Before you can generate tests for a module, it must have at least one version published. Tests can only be generated once per module and are intended to be reviewed by the module's authors before being checked into version control and maintained with the rest of the module's content. If the module's configuration files exceed 128KB in total size, Terraform Cloud will not be able to generate tests for that module.

You must have [permission to manage registry modules](/terraform/cloud-docs/users-teams-organizations/permissions#manage-private-registry) and [permission to manage module test generation](/terraform/cloud-docs/users-teams-organizations/permissions#manage-module-test-generation-beta) in order to generate tests.
```